torch/_inductor/codecache.py

```diff
@@ -154,6 +154,10 @@ def get_cpp_wrapper_cubin_path_name() -> str:
     return "cubin_path" if torch.version.hip is None else "hsaco_path"
 
 
+def get_actual_cubin_format() -> str:
+    return "cubin" if torch.version.hip is None else "hsaco"
+
```

Comment:
What about xpu's spv format? I noticed [this piece of code](https://www.internalfb.com/code/fbsource/[076a4646d3a5]/fbcode/caffe2/torch/_inductor/runtime/triton_heuristics.py?lines=919-921) in triton_heuristics -- could we add the `bin_type` to the params instead and then you can just get the cubin format from params?

Reply:
The function is currently only used in test_compile_after_package. Writing to CudaKernelParamCache will not work because it's not available at the test time.

I can extend this function to take a device input, to make it compatible for xpu.